<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="ValidationException.xml" source-language="en-US" target-language="zh-TW">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5e7f076d39684afc47faa3f7aac66f5782924a98a.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">e7f076d39684afc47faa3f7aac66f5782924a98a</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>Represents the exception that occurs during validation of a data field when the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> class is used.</source>
          <target state="translated">表示使用 <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> 類別驗證資料欄位時發生的例外狀況 (Exception)。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>A validation exception occurs if an input value does not match the expected data type, range or pattern of the data field.</source>
          <target state="translated">如果輸入的值不符合預期的資料類型、 範圍或資料欄位的模式，就會發生驗證例外狀況。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>For example, if a user enters an integer value in a data field that expects a <ph id="ph1">&lt;xref:System.Data.SqlDbType.DateTime&gt;</ph> value, a validation exception occurs.</source>
          <target state="translated">例如，如果使用者輸入的整數值的資料欄位中，必須要有<ph id="ph1">&lt;xref:System.Data.SqlDbType.DateTime&gt;</ph>值，就會發生驗證例外狀況。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>This class uses the <ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationAttribute&gt;</ph> class to customize validations.</source>
          <target state="translated">這個類別會使用<ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationAttribute&gt;</ph>類別來自訂驗證。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>A <ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationException&gt;</ph> is thrown if a validation error occurs.</source>
          <target state="translated">A<ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationException&gt;</ph>如果發生驗證錯誤時，會擲回。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>The exception is thrown when the <ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationAttribute.Validate%2A&gt;</ph> method is called.</source>
          <target state="translated">擲回例外狀況時<ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationAttribute.Validate%2A&gt;</ph>方法呼叫。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>All the exceptions that are thrown during validation are contained in the <ph id="ph1">&lt;xref:System.Web.DynamicData.IDynamicValidatorException.InnerExceptions%2A&gt;</ph> collection.</source>
          <target state="translated">中所包含的所有驗證期間所擲回的例外狀況<ph id="ph1">&lt;xref:System.Web.DynamicData.IDynamicValidatorException.InnerExceptions%2A&gt;</ph>集合。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>You can retrieve each validation exception by iterating through the <ph id="ph1">&lt;xref:System.Web.DynamicData.IDynamicValidatorException.InnerExceptions%2A&gt;</ph> collection.</source>
          <target state="translated">您可以擷取每個驗證例外狀況逐一<ph id="ph1">&lt;xref:System.Web.DynamicData.IDynamicValidatorException.InnerExceptions%2A&gt;</ph>集合。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="T:System.ComponentModel.DataAnnotations.ValidationException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> class.</source>
          <target state="translated">初始化 <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> class using an error message generated by the system.</source>
          <target state="translated">使用系統產生的錯誤訊息，初始化 <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor">
          <source>This method is the default constructor for the <ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationException&gt;</ph> class.</source>
          <target state="translated">這個方法是預設建構函式<ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationException&gt;</ph>類別。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String)">
          <source>A specified message that states the error.</source>
          <target state="translated">陳述錯誤的指定訊息。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> class using a specified error message.</source>
          <target state="translated">使用指定的錯誤訊息，初始化 <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String)">
          <source>For more information about how to use this constructor, see <bpt id="p1">[</bpt>How to: Customize Data Field Validation in the Data Model<ept id="p1">](http://msdn.microsoft.com/library/47d3b04e-1ab5-466d-bc58-92f163f3d7f5)</ept>.</source>
          <target state="translated">如需如何使用這個建構函式的詳細資訊，請參閱<bpt id="p1">[</bpt>How to： 資料模型中的自訂資料欄位驗證<ept id="p1">](http://msdn.microsoft.com/library/47d3b04e-1ab5-466d-bc58-92f163f3d7f5)</ept>。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The object that holds the serialized data.</source>
          <target state="translated">存放序列化資料的物件。</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Context information about the source or destination of the serialized object.</source>
          <target state="translated">關於序列化物件之來源或目的端的內容資訊。</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> class using serialized data.</source>
          <target state="translated">使用序列化資料，初始化 <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This constructor is called during deserialization to reconstitute the exception object that is transmitted over a stream.</source>
          <target state="translated">重新構成經由資料流傳輸的例外狀況物件的還原序列化期間呼叫這個建構函式。</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.Exception)">
          <source>The error message.</source>
          <target state="translated">錯誤訊息。</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.Exception)">
          <source>The collection of validation exceptions.</source>
          <target state="translated">驗證例外狀況的集合。</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> class using a specified error message and a collection of inner exception instances.</source>
          <target state="translated">使用指定的錯誤訊息和內部例外狀況執行個體集合，初始化 <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.Exception)">
          <source>You typically will not use this constructor because validation exceptions are stored in the <ph id="ph1">&lt;xref:System.Web.DynamicData.IDynamicValidatorException.InnerExceptions%2A&gt;</ph> property, not <ph id="ph2">`innerException`</ph>.</source>
          <target state="translated">您通常不會使用這個建構函式因為驗證例外狀況會儲存在<ph id="ph1">&lt;xref:System.Web.DynamicData.IDynamicValidatorException.InnerExceptions%2A&gt;</ph>屬性，不<ph id="ph2">`innerException`</ph>。</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.ComponentModel.DataAnnotations.ValidationResult,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>The list of validation results.</source>
          <target state="translated">驗證結果的清單。</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.ComponentModel.DataAnnotations.ValidationResult,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>The attribute that caused the current exception.</source>
          <target state="translated">造成目前例外狀況的屬性。</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.ComponentModel.DataAnnotations.ValidationResult,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>The value of the object that caused the attribute to trigger the validation error.</source>
          <target state="translated">造成此屬性觸發驗證錯誤的物件值。</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.ComponentModel.DataAnnotations.ValidationResult,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> class by using a validation result, a validation attribute, and the value of the current exception.</source>
          <target state="translated">使用驗證結果、驗證屬性以及目前例外狀況的值，初始化 <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>The message that states the error.</source>
          <target state="translated">陳述錯誤的訊息。</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>The attribute that caused the current exception.</source>
          <target state="translated">造成目前例外狀況的屬性。</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>The value of the object that caused the attribute to trigger validation error.</source>
          <target state="translated">造成此屬性觸發驗證錯誤的物件值。</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> class using a specified error message, a validation attribute, and the value of the current exception.</source>
          <target state="translated">使用指定的錯誤訊息、驗證屬性 (Attribute) 以及目前例外狀況的值，初始化 <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ComponentModel.DataAnnotations.ValidationException.#ctor(System.String,System.ComponentModel.DataAnnotations.ValidationAttribute,System.Object)">
          <source>For more information, see How to: Customize Data Field Validation in the Data Model.</source>
          <target state="translated">如需詳細資訊，請參閱 &lt; 如何： 自訂資料欄位驗證資料模型中。</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" uid="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationAttribute">
          <source>Gets the instance of the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> class that triggered this exception.</source>
          <target state="translated">取得觸發此例外狀況之 <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> 類別的執行個體。</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationAttribute">
          <source>An instance of the validation attribute type that triggered this exception.</source>
          <target state="translated">觸發此例外狀況之驗證屬性型別的執行個體。</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationAttribute">
          <source>This type is used to enforce validation based on the metadata associated with the data table.</source>
          <target state="translated">此類型用來強制執行驗證，根據與資料表相關聯的中繼資料。</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationAttribute">
          <source>If an error occurs, the <ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationAttribute&gt;</ph> class triggers a <ph id="ph2">&lt;xref:System.ComponentModel.DataAnnotations.ValidationException&gt;</ph>.</source>
          <target state="translated">如果發生錯誤，<ph id="ph1">&lt;xref:System.ComponentModel.DataAnnotations.ValidationAttribute&gt;</ph>類別觸發程序<ph id="ph2">&lt;xref:System.ComponentModel.DataAnnotations.ValidationException&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="136" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationAttribute">
          <source>This class can be overridden to create custom validation attributes.</source>
          <target state="translated">這個類別會覆寫來建立自訂驗證屬性。</target>       </trans-unit>
        <trans-unit id="137" translate="yes" xml:space="preserve" uid="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationResult">
          <source>Gets the <ph id="ph1">&lt;see cref="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationResult" /&gt;</ph> instance that describes the validation error.</source>
          <target state="translated">取得描述驗證錯誤的 <ph id="ph1">&lt;see cref="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationResult" /&gt;</ph> 執行個體。</target>       </trans-unit>
        <trans-unit id="138" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationResult">
          <source>The <ph id="ph1">&lt;see cref="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationResult" /&gt;</ph> instance that describes the validation error.</source>
          <target state="translated">描述驗證錯誤的 <ph id="ph1">&lt;see cref="P:System.ComponentModel.DataAnnotations.ValidationException.ValidationResult" /&gt;</ph> 執行個體。</target>       </trans-unit>
        <trans-unit id="139" translate="yes" xml:space="preserve" uid="P:System.ComponentModel.DataAnnotations.ValidationException.Value">
          <source>Gets the value of the object that causes the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> class to trigger this exception.</source>
          <target state="translated">取得造成 <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> 類別觸發此例外狀況之物件的值。</target>       </trans-unit>
        <trans-unit id="140" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.ComponentModel.DataAnnotations.ValidationException.Value">
          <source>The value of the object that caused the <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> class to trigger the validation error.</source>
          <target state="translated">造成 <ph id="ph1">&lt;see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /&gt;</ph> 類別觸發驗證錯誤之物件的值。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>